src/NUnitFramework/framework/Internal/PlatformHelper.cs

```diff
@@ -245,6 +246,13 @@ public bool IsPlatformSupported( string platform )
                 case "WIN8.1":
                     isSupported = os.IsWindows81;
                     break;
+                case "WINDOWS10":
+                case "WIN10":
+                    isSupported = os.IsWindows10;
+                    break;
+                case "WINDOWSSERVER":
```

Comment:
The three new OS strings will need to be documented. I will do that after the review, I just wanted to make sure people are happy with the choices. WindowsServer seems odd to me, but it is the official name as far as I can tell, no versioning.